cg/meta/report/mip_dna.py

```diff
@@ -49,7 +49,9 @@ def get_sample_metadata(
         return MipDNASampleMetadataModel(
             bait_set=self.lims_api.capture_kit(sample.internal_id),
             gender=parsed_metrics.predicted_sex,
-            million_read_pairs=round(sample.reads / 2000000, 1) if sample.reads else None,
+            million_read_pairs=round(sample.reads / 2000000, 1)
+            if sample.reads or isinstance(sample.reads, int)
+            else None,
```

Comment:
Looks like this code block is repeated here and in cg/meta/report/balsamic.py. Let's make it a united tested function instead and reuse it.

Reply:
Good idea!